src/plugins/intel_gpu/src/runtime/shape_predictor.cpp

```diff
@@ -56,25 +56,31 @@ bool ShapePredictor::can_preallocate(size_t desired_buffer_size) {
     return device_mem_usage + desired_buffer_size < _engine->get_device_info().max_global_mem_size * memory_threshold;
 }
 
-std::pair<bool, ov::Shape> ShapePredictor::predict_preallocation_shape(const std::string& id,
+std::pair<bool, ov::Shape> ShapePredictor::predict_preallocation_shape(const std::string& orig_id,
```

Comment:
@sshlyapn Maybe we should replace std::string here with unique node id (size_t) ?

Reply:
Can we refactor it in a separate PR?